pageserver/src/tenant/timeline.rs

```diff
@@ -3892,9 +3962,25 @@ impl Timeline {
 
         for partition in partitioning.parts.iter() {
             let img_range = start..partition.ranges.last().unwrap().end;
-            if !force && !self.time_for_new_image_layer(partition, lsn).await {
-                start = img_range.end;
-                continue;
+
+            if partition.overlaps(&Key::metadata_key_range()) {
+                // The next patch will correctly create image layers for metadata keys, and it would be a
```

Comment:
Can you turn the wording into a `TODO` instead?

Reply:
fixed in https://github.com/neondatabase/neon/pull/7503/commits/804ea39e084fc70596a56ad141380dd92a453b0c